src/shogun/evaluation/CrossValidation.cpp

```diff
@@ -16,31 +16,29 @@
 #include <shogun/lib/observers/ObservedValueTemplated.h>
 #include <shogun/machine/Machine.h>
 #include <shogun/mathematics/Statistics.h>
-#include <shogun/util/converters.h>
+#include <shogun/lib/View.h>
 
 using namespace shogun;
 
-CCrossValidation::CCrossValidation() : CMachineEvaluation()
+CCrossValidation::CCrossValidation() : RandomMixin<CMachineEvaluation>()
```

Comment:
@theartful @vigsterkr question (also see email): is there a better way to allow to set the seed on the `CrossValidation` object? It doesnt use/generate random numbers itself, but the recursive setting is really helpful here (otherwise user needs to set multiple seeds in the parameters)